packages/realm/src/Configuration.ts

```diff
@@ -47,6 +50,15 @@ export type BaseConfiguration = {
   disableFormatUpgrade?: boolean;
   encryptionKey?: ArrayBuffer | ArrayBufferView | Int8Array;
   onMigration?: MigrationCallback;
+  /**
+   * A flag to indicate whether Realm should resolve
+   * embedded object constraints after migration. If this is `true` then all embedded objects
+   * without a parent will be deleted and every embedded object with multiple references to it
+   * will be duplicated so that every referencing object will hold its own copy of the
+   * embedded object.
+   * @default false
+   */
```

Comment:
I'm noticing that this seems to be the only configuration option that is documented.  I believe we need a follow-up ticket to document the realm configuration in code.

Reply:
Yep, I noticed too. I have created https://github.com/realm/realm-js/issues/6081